desktop/cmp/panel/Panel.js

```diff
@@ -147,6 +149,16 @@ export const [Panel, panel] = hoistCmp.withFactory({
 
 });
 
+function parseError(error, contextModel) {
+    if (error === 'lastLoadException') {
+        const {lastLoadException} = contextModel;
+        return lastLoadException ? errorMessage({error: lastLoadException}) : null;
+    }
+    if (isString(error))        return errorMessage({error});
+    if (isValidElement(error))  return error;
```

Comment:
We should be able to take an actual `Error` as well - I can imagine a model choosing to save a reference to an actual error, distinct from loadSupport (i.e. it might not be implementing `doLoadAsync()`).

Reply:
agree -- think we could actually just pass everything to errorMessage -- that would be the most straightforward.  But the issue of customization is something we need to discuss (see below)